pwnlib/tubes/ssh.py

```diff
@@ -402,8 +400,8 @@ def getenv(self, variable, **kwargs):
                            'print(getenv(%r))' % variable,))
 
         try:
-            with context.local(log_level='error'):
-                python = self.parent.which('python')
+            with context.quiet:
+                python = self.parent.which('python2.7') or self.parent.which('python')
```

Comment:
Why not prefer python3? :c

Reply:
Because I'm not sure it'll work with Python3 yet, but I *AM* sure that some distros have the `python` binary set to Py3 now.  Just being cautious, particularly for Ubuntu 20.04 and newer.